src/Ormolu/Printer/Meat/ImportExport.hs

```diff
@@ -36,9 +36,7 @@ p_hsmodImport ImportDecl {..} = do
   space
   when ideclQualified (txt "qualified")
   space
-  case ideclPkgQual of
-    Nothing -> return ()
-    Just slit -> atom slit
+  forM_ ideclPkgQual atom
```

Comment:
Meh. I'd argue this one should just be disabled in `.hlint.yaml`.

Reply:
Reverted.

Should `.hlint.yaml` be added?

Reply:
I would add it, yeah. Once I start using HLint, I usually end up needing a `.hlint.yaml` almost immediately.

Reply:
If this should be done, this should be done a separate PR.

Reply:
`.hlint.yaml` should be added if we decide to run Hlint on CI. Otherwise no.